spring-kafka-4/pom.xml

```diff
@@ -31,7 +31,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+        </dependency>
```

Comment:
We have a guideline:
"if the post is not specifically about Lombok, then let’s avoid adding it to a project"

Reply:
@theangrydev actually this is part of another PR to move the code to this module as asked by Jira support, it's coming from my master branch. 
https://jira.baeldung.com/browse/BAEL-6420
https://github.com/eugenp/tutorials/pull/18562 
this should be merged first

Reply:
But yes, i will remove it